### DISCUSSION TITLE
V3 - How do you retrieve orders that are open?

### DISCUSSION THREAD

**Discussion by AndrewLeonard** [etsy/open-api#804] [Q&A] [answered]
Is there something similar to V2's "open" in V3?

**Comment by Gareth064** [comment, marked as answer]
Look at the list of parameters you can send with the get shop receipts endpoint. I believe there is one called was_shipped. If you set that to false then I believe you can get open orders.

**Reply by AndrewLeonard** [reply]
Hi Gareth,

Thanks for that - I ended up using the combination of "was_shipped" (false) and "paid" (true).

**Reply by Gareth064** [reply]
Glad I could help. If you believe you are sorted then please mark this questioned as Answered 👍
